utils/dmidecode.go

```diff
@@ -1,8 +1,10 @@
 package utils
 
 import (
+	"cmp"
```

Comment:
TIL: cmp

Reply:
Yep cmp.Or is new in 1.22, looking forward to making more use of it!